### PR TITLE
Testing in Jenkins

### PR DESCRIPTION
**Pull request by fstagni**
On top of https://github.com/DIRACGrid/Pilot/pull/176

Old schema looks OK, new one fails with:
```
+ python dirac-pilot.py -M 1 -S DIRAC-Certification -N jenkins.cern.ch -Q jenkins-queue_not_important -n DIRAC.Jenkins.ch --cert --certLocation=/home/dirac/certs --debug
Traceback (most recent call last):
  File "dirac-pilot.py", line 52, in <module>
    pilotParams = PilotParams()
  File "/home/dirac/scratch/workspace/Pilot3_CVM4_pipeline/PilotInstallDIR/pilotTools.py", line 770, in __init__
    self.__initJSON2()
  File "/home/dirac/scratch/workspace/Pilot3_CVM4_pipeline/PilotInstallDIR/pilotTools.py", line 941, in __initJSON2
    self.commands = [elem.strip() for elem in commands.split(",")]
AttributeError: 'list' object has no attribute 'split'
```
(see https://jenkins-dirac.web.cern.ch/view/DIRAC/job/Pilot3_CVM4_pipeline/359/console)

### CODE REVIEW

**Comment by martynia** [comment]
Hi,
A change of` br` to` r` in `proxyTools`  causes my tests to fail for python 3 fail with an error:

>       return _compile(pattern, flags).findall(string)
E       TypeError: cannot use a string pattern on a bytes-like object

`br` is harmless for Python2 on linux, I believe.

**Comment by fstagni** [comment]
`br` was giving me a SyntaxError on py2.

**Comment by martynia** [comment]
Are you sure you tested that with br and not rb ? I have a unit test which uses br passes both on py2 and py3 (these test do pass on GitHub btw)…but I had a problem with a workflow containing black.

I typed: br
Black changed it to rb (on save). - and this is not legal in py2, OK in py3

So I had to eliminate black from the workflow.

Please double check if your system does not suffer from this.
 Cheers, JM

**Comment by fstagni** [comment]
Ah, possible. Let me try again.

**Comment by fstagni** [comment]
> Ah, possible. Let me try again.

Indeed, it is OK now.
